SpatialGDK/Source/SpatialGDK/Public/Interop/GlobalStateManager.h

```diff
@@ -40,6 +40,8 @@ class SPATIALGDK_API UGlobalStateManager : public UObject
 	void ApplyVirtualWorkerMappingFromQueryResponse(const Worker_EntityQueryResponseOp& Op);
 	void ApplyDeploymentMapDataFromQueryResponse(const Worker_EntityQueryResponseOp& Op);
 
+	void QueryTranslation();
```

Comment:
I think this function and `ApplyVirtualWorkerMappingFromQueryResponse` can both be const, not a big deal though

Reply:
QueryTranslation can't now that I've got an in flight bool. I'll change the other.